### ISSUE TITLE
training loss curve

### ISSUE THREAD

**Issue by pangxincheng** [RoboFlamingo/RoboFlamingo#4]
Hi, thanks for your great work!
Can you provide the loss curve during the training process, or explain the approximate range of loss values during the training process? I would like to confirm the correctness of my training process.
Thanks!

**Comment by lixinghang12** [comment]
@pangxincheng Sure. Here is a log file of mse and bce loss for training the MPT-3B-IFT on ABCD->D split.
[output.log](https://github.com/RoboFlamingo/RoboFlamingo/files/13498984/output.log)
